slsim/lens.py

```diff
@@ -245,6 +249,12 @@ def validity_test(
             magnitude limits of integrated lensed arc
         :type mag_arc_limit: dict with key of bands and values of
             magnitude limits
+        :param second_bright_image_cut: Dictionary containing maximum
+            magnitude of the second brightest image and corresponding
+            band. If provided, selects lenses where the second brightest
+            image has a magnitude less than or equal to provided
+            magnitude. eg: second_bright_image_cut = {"band": "i",
+            "second_bright_mag_max": 23}
```

Comment:
I don't think you need to name the parameter itself "second_bright_mag_max" since the entire dictionary is already labeled this way. Perhaps simply "mag_max" or something like that.